vita3k/gui/src/controllers_dialog.cpp

```diff
@@ -282,9 +283,58 @@ void draw_controllers_dialog(GuiState &gui, EmuEnvState &emuenv) {
                             ImGui::EndTable();
                         }
                         ImGui::EndTable();
+                    }
 
-                        ImGui::End();
+                    const SDL_JoystickGUID guid = SDL_JoystickGetDeviceGUID(i);
+                    if (ctrl.controllers[guid].has_led) {
+                        const auto set_color_led = [&](const std::vector<int> led) {
```

Comment:
```suggestion
                        const auto set_color_led = [&](const std::vector<int>& led) {
```

Reply:
For that 🤔 why adding &, value is just sended 🤷‍♂️

Reply:
Done